vnext/Microsoft.ReactNative.Xaml/FabricXamlControl.h

```diff
@@ -0,0 +1,259 @@
+#pragma once
+
```

Comment:
We can probably get most of this file codegen'd from a spec file.
Can we create a FabricXamlNativeComponent.js?
Or do we want a separate spec for each Xaml Component.... - We probably do since they will all have different properties.

then the M.RN.Xaml dll would provide the helpers to make it easy to implement those components.